bin/i18n/sync-in.rb

```diff
@@ -73,8 +73,9 @@ def get_i18n_strings(level)
 
       ## Function Names
       functions = blocks.xpath("//block[@type=\"procedures_defnoreturn\"]")
-      i18n_strings['function_names'] = Hash.new unless functions.empty?
-      functions.each do |function|
+      behaviors = blocks.xpath("//block[@type=\"behavior_definition\"]")
+      i18n_strings['function_names'] = Hash.new unless functions.empty? && behaviors.empty?
```

Comment:
is there a specific reason to combine function and behavior names like this?

Reply:
Mostly to avoid adding another file. I'm not particularly attached though so I can create a new `behaviors.yml` if you think it's cleaner?

Reply:
I'd prefer to either come up with a name that makes it clear this is a category that includes both functions and behaviors, or add behaviors separately alongside functions

Reply:
Looks like our `sync-out` will handle a new file pretty robustly so I'll just create a new file